Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -163,6 +163,11 @@ open class ComposerVC: _ViewController,
         channelConfig?.commands.isEmpty == false
     }
 
+    /// A Boolean value indicating whether the user mentions are enabled.
+    open var isMentionsEnabled: Bool {
+        true
+    }
```

Comment:
What if users just want to assign this, in their MessageList subclass? IMHO better to have this get/set and not overridable

Reply:
As mentioned above the `ComposerVC.isCommandsEnabled` and `ComposerVC.isAttachmentsEnabled` are also overridable properties, so it makes sense for this one to be as well. These two are really useful to be computed properties because by default they read from the feature flags on the channel config. So, it is way to override that behaviour. Either way, for consistency at least on the `ComposerVC` component we should keep it like this.